### PR TITLE
Fix https://beta.monarchinitiative.org/about/data-sources page (currently blank)

### PR DESCRIPTION
**Pull request by justaddcoffee**
Remove redundant 'sources' key causing blank data-sources page

### CODE REVIEW

File: src/api/Sources.js [hunk 1]
```diff
@@ -256,5 +256,5 @@ export default function getSourceInfo() {
       monarchUsage: "Monarch stores gene to phenotype associations and gene function data",
       vocabulary: "APO, RO"
     }
-  ];
+  ]
```
Comment (kshefchek) [inline src/api/Sources.js]: The linter complains because we're using it as javascript even though it's mostly just json, but either way is fine

Comment (justaddcoffee) [reply src/api/Sources.js]: Thanks Kent - what link error are you seeing? This seems to pass my linter:

```
$ vue-cli-service lint --no-fix src/api/Sources.js 

DONE  No lint errors found!
```

(Also what linter should I be using?)

Comment (kshefchek) [reply src/api/Sources.js]: odd, I get:

    error: Missing semicolon (semi) at src/api/Sources.js:259:4:
      257 |       vocabulary: "APO, RO"
      258 |     }
    > 259 |   ]
          |    ^
      260 | }
      261 | 

I'm running npm run lint which calls vue-cli-service lint --no-fix, so should be the same, node version 8.12.0

Comment (justaddcoffee) [reply src/api/Sources.js]: Oh, sorry!  I saw that error too, fixed it, and forgot (thus no error for me). Should be linted now if you do a pull

**Comment by monicacecilia** [comment]
@kshefchek - ping! 
Is this ready to be merged?

**Comment by kshefchek** [comment]
Looks good to me merge away

**Pull request merged**